### PR TITLE
Updated schema

### LINKED ISSUE #16: feature request: expose package license
It would be useful to be able to access a packages license name (the way pub.dev webpage for a package shows it), eg. "MIT".

### PR DESCRIPTION
**Pull request by leoafarias**
Closes #16 and #43

### CODE REVIEW

File: lib/src/models/package_score_model.dart [hunk 1]
```diff
@@ -1,144 +1,25 @@
-import 'package:collection/collection.dart';
+import 'package:dart_mappable/dart_mappable.dart';
 
-// ignore_for_file: lines_longer_than_80_chars
+part 'package_score_model.mapper.dart';
 
-/// Package Score Model
-class PackageScore {
-  final int? grantedPoints;
```
Comment (Rexios80) [inline lib/src/models/package_score_model.dart:7]: @leoafarias Why were these changed to non-nullable? Are we sure these are never null?

Comment (leoafarias) [reply lib/src/models/package_score_model.dart:7]: Yes they come nullable if the package is processing.

Comment (Rexios80) [reply lib/src/models/package_score_model.dart:7]: Shouldn't they be changed back to nullable then?

Comment (leoafarias) [reply lib/src/models/package_score_model.dart:7]: Sorry for the confusion. They used to be nullable before, but on my tools, they are not anymore. Let me know if you are seeing something different.

Comment (Rexios80) [reply lib/src/models/package_score_model.dart:7]: Did you test with a package that was in the middle of scanning? What are the values?
